test/integration/targets/dnf/tasks/main.yml

```diff
@@ -59,10 +59,9 @@
 
 - include_tasks: modularity.yml
   when:
-    - astream_name is defined
```

Comment:
It appears that this condition effectively disabled testing modularity on all targets because `astream_name` is only defined inside `modularity.yml`.

Reply:
Good catch.